virttest/remote.py

```diff
@@ -67,7 +67,7 @@ def ssh_login_to_migrate(client, host, port, username, password, prompt, linesep
                             linesep, log_filename, None, timeout,
                             interface, identity_file, status_test_command,
                             verbose, bind_ip)
-
+    verbose = "-vv" if verbose else ""
     cmd = ("ssh %s -o UserKnownHostsFile=%s "
            "-o StrictHostKeyChecking=no -p %s" %
            (verbose, user_known_hosts_file, port))
```

Comment:
How about not dynamically changing the type of this variable and using something simpler:
```diff
    cmd = ("ssh %s -o UserKnownHostsFile=%s "
           "-o StrictHostKeyChecking=no -p %s" %
-           (verbose, user_known_hosts_file, port))
+           ("-vv" if verbose else "", user_known_hosts_file, port))
```

Reply:
Updated.